core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.component.spec.ts

```diff
@@ -396,6 +396,41 @@ describe('Admin dev mode activities tab', () => {
     }));
   });
 
+  describe('.generateNewClassroom', () => {
+    it('should generate classroom data', async(() => {
+      spyOn(adminBackendApiService, 'generateDummyClassroomDataAsync')
+        .and.returnValue(Promise.resolve());
+      spyOn(component.setStatusMessage, 'emit');
+
+      component.generateNewClassroom();
+
+      expect(component.setStatusMessage.emit)
+        .toHaveBeenCalledWith('Processing...');
+
+      fixture.whenStable().then(() => {
```

Comment:
Please add a comment explaining why you used `fixture.whenStable()` here.

Reply:
Added a comment